resources/views/pages/saved-items.blade.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * This file (profile.blade.php) was created on 06/19/2016 at 16:17.
+ * (C) Max Cassee
+ * This project was commissioned by HU University of Applied Sciences.
+ */
+?>
+@extends('layout.HUdefault')
+@section('title')
+    Saved items
+@stop
+@section('content')
+<?php
+use App\Student;
+use App\SavedLearningItem
+/** @var Student $student */;
+/** @var SavedLearningItem $sli */?>
+
+    <div class="container-fluid">
+        <div class="row">
+            <!-- Profile Info -->
+            <div class="col-md-3">
+            
+
+                @card
+                    <h1>{{ __('saved_learning_items.saved') }}</h1>
+                    <h2>{{ __('saved_learning_items.timeline') }}</h2>
+                    
+                    @foreach($sli as $item)
+                    @if($item->category == 'tip')
+                        @foreach($tips as $tip)
+                        @if($tip->id == $item->item_id)
+                        <div class="alert" style="background-color: #00A1E2; color: white; margin-left:2px; margin-bottom: 10px"
+                             role="alert">
+                            <h4>{{ __('tips.personal-tip') }}</h4>
+                            <p>{!! nl2br($tip->tipText) !!}</p>
```

Comment:
The `tipText` property of `Tip` is a "raw" value. It needs to be processed before it is shown to the user, to include the calculated values.
For example, now this code gives the following:
<img width="385" alt="image" src="https://user-images.githubusercontent.com/4541046/70443190-03d71580-1a98-11ea-8284-358692325b12.png">

On the homepage this is done in the `ApplicableTipFetcher`, which checks what tips are applicable to the student. In its [`fetchForCohort` method](https://github.com/HUInstituteForICT/workplacelearning/blob/ef8e0d685be3bf434e9b037c83765934545a9d08/app/Tips/Services/ApplicableTipFetcher.php#L31-L35) it does so by using the `TipEvaluator`. You could pass an instance of the `TipEvaluator` to the view, and process the saved tip just before displaying its text. Note, you need to use the result of the evaluator, which returns an `EvaluatedTip`. Use its `getTipText` to access the processed text.